src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -356,20 +351,16 @@ namespace HVACUnitaryBypassVAV {
         // find the number of each type of CBVAV unit
         std::string CurrentModuleObject = "AirLoopHVAC:UnitaryHeatCool:VAVChangeoverBypass";
 
-        auto &CBVAV(state.dataHVACUnitaryBypassVAV->CBVAV);
-
         // Update Num in state and make local convenience copy
         int NumCBVAV = state.dataHVACUnitaryBypassVAV->NumCBVAV =
             state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, CurrentModuleObject);
 
         // allocate the data structures
-        CBVAV.allocate(NumCBVAV);
+        state.dataHVACUnitaryBypassVAV->CBVAV.resize(NumCBVAV);
```

Comment:
I was getting a failure in debug and had to change this to resize. This happened in the middle of the Type refactors.